esp_crt_bundle/cacrt_all.pem

```diff
@@ -0,0 +1,3460 @@
+##
+## Bundle of CA Root Certificates
+##
+## Certificate data from Mozilla as of: Tue Jul 19 03:12:06 2022 GMT
```

Comment:
Is there a fresher bundle available?

Reply:
Yes, I updated using the version from 2023-01-10